Job invocation: ValueError: not enough values to unpack (expected 2, got 1)

When running a job invocation from Robottelo (e.g. `tests/foreman/cli/test_remoteexecution.py`), I get:
```
self = <tests.foreman.cli.test_remoteexecution.TestRemoteExecution object at 0x7fb2633723d0>

    @pytest.mark.tier3
    @pytest.mark.upgrade
    def test_positive_run_receptor_installer(self):
        """Run Receptor installer ("Configure Cloud Connector")
    
        :CaseComponent: RHCloud-CloudConnector
    
        :id: 811c7747-bec6-1a2d-8e5c-b5045d3fbc0d
    
        :expectedresults: The job passes, installs Receptor that peers with c.r.c
    
        :BZ: 1818076
        """
        # Set Host parameter source_display_name to something random.
        # To avoid 'name has already been taken' error when run multiple times
        # on a machine with the same hostname.
        host_id = Host.info({'name': settings.server.hostname})['id']
        Host.set_parameter(
            {'host-id': host_id, 'name': 'source_display_name', 'value': gen_string('alpha')}
        )
    
        template_name = 'Configure Cloud Connector'
>       invocation = make_job_invocation(
            {
                'async': True,
                'job-template': template_name,
                'inputs': 'satellite_user="{}",satellite_password="{}"'.format(
                    settings.server.admin_username, settings.server.admin_password
                ),
                'search-query': f"name ~ {settings.server.hostname}",
            }
        )

tests/foreman/cli/test_remoteexecution.py:516: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
robottelo/decorators/__init__.py:113: in cacheable_function
    new_object = func(options)
robottelo/cli/factory.py:1030: in make_job_invocation
    return create_object(JobInvocation, args, options)
robottelo/cli/factory.py:134: in create_object
    result = cli_object.create(options)
robottelo/cli/base.py:150: in create
    new_obj = cls.info(info_options)
robottelo/cli/base.py:293: in info
    result = hammer.parse_info(result)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

output = ['---', 'ID: 10', 'Description: Configure Cloud Connector', 'Status: running', 'Success: 0', 'Failed: 0', ...]

    def parse_info(output):
        """Parse the info output and returns a dict mapping the values."""
        # info dictionary
        contents = {}
        sub_prop = None  # stores name of the last group of sub-properties
        sub_num = None  # is not None when list of properties
        second_level_key = None  # is set when a possible second level is detected
    
        for line in output:
            # skip empty lines
            if line == '':
                continue
            current_indent_level = get_line_indentation_level(line)
            if current_indent_level <= 1:
                # we are entering or leaving a second level from lower/upper levels
                # clear the second level key
                second_level_key = None
            if line.startswith(' '):  # sub-properties are indented
                # values are separated by ':' or '=>', but not by '::' which can be
                # entity name like 'test::params::keys'
                if line.find(':') != -1 and not line.find('::') != -1:
                    key, value = line.lstrip().split(":", 1)
                elif line.find('=>') != -1 and len(line.lstrip().split(" =>", 1)) == 2:
                    key, value = line.lstrip().split(" =>", 1)
                else:
                    key = value = None
    
                if key is None and value is None:
                    # Parse single attribute collection properties
                    # Template
                    #  1) template1
                    #  2) template2
                    #
                    # or
                    # Template
                    #  template1
                    #  template2
                    match = re.match(r'\d+\)\s+(.+)$', line.lstrip())
    
                    if match is None:
                        match = re.match(r'(.*)$', line.lstrip())
    
                    value = match.group(1)
    
                    # adding list to 1 level, for example:
                    # {'template': ['template1', 'template2']}
                    if isinstance(contents[sub_prop], dict) and not contents[sub_prop]:
                        contents[sub_prop] = []
                        contents[sub_prop].append(value)
                    elif isinstance(contents[sub_prop], list):
                        contents[sub_prop].append(value)
                    else:
                        # adding list to 2 level, for example:
                        # {'subscription-information':
                        #      {'registered-by-activation-keys': ['ak1', 'ak2']}
                        #  }
                        last_key = list(contents[sub_prop].keys())[-1]
                        if not contents[sub_prop][last_key]:
                            contents[sub_prop][last_key] = [value]
                        else:
                            contents[sub_prop][last_key].append(value)
                else:
                    # some properties have many numbered values
                    # Example:
                    # Content:
                    #  1) Repo Name: repo1
                    #     URL:       /custom/4f84fc90-9ffa-...
                    #  2) Repo Name: puppet1
                    #     URL:       /custom/4f84fc90-9ffa-...
                    starts_with_number = re.match(r'(\d+)\)', key)
                    if starts_with_number:
                        sub_num = int(starts_with_number.group(1))
                        # no. 1) we need to change dict() to list()
                        if sub_num == 1:
                            contents[sub_prop] = []
                        # remove number from key
                        key = re.sub(r'\d+\)', '', key)
                        # append empty dict to array
                        contents[sub_prop].append({})
    
                    key = key.lstrip().replace(' ', '-').lower()
                    value = value.lstrip()
                    # add value to dictionary
                    if sub_num is not None:
                        contents[sub_prop][-1][key] = value
                    else:
                        # a third level is always represented as a dictionary and
                        # we need to detect if we are at third level
                        # example:
                        # Content Information:
                        #     Content View:
                        #         ID:   10
                        #         Name: Default Organization View
                        # the "ID" and "Name" are located at third indent level
                        # "content view" is located at second indent level
                        if current_indent_level == 2 and second_level_key:
                            # we are at third level indentation
                            if not contents[sub_prop][second_level_key]:
                                contents[sub_prop][second_level_key] = {}
                            contents[sub_prop][second_level_key][key] = value
                        else:
                            contents[sub_prop][key] = value
                        if current_indent_level == 1 and not value:
                            # always set the last possible second level key
                            # that can form a third level
                            second_level_key = key
            else:
                sub_num = None  # new property implies no sub property
>               key, value = line.lstrip().split(":", 1)
E               ValueError: not enough values to unpack (expected 2, got 1)

robottelo/cli/hammer.py:305: ValueError
```
First item of list `output` contains '---'.
Happens on Sat 6.9.

--- a/robottelo/cli/hammer.py
+++ b/robottelo/cli/hammer.py
@@ -203,8 +203,8 @@ def parse_info(output):
     second_level_key = None  # is set when a possible second level is detected
 
     for line in output:
-        # skip empty lines
-        if line == '':
+        # skip empty lines and dividers
+        if line == '' or line == '---':
             continue
         current_indent_level = get_line_indentation_level(line)
         if current_indent_level <= 1:

